### PR TITLE
fixes #5812 - url parameter in compute_resource#create is not required for EC2

### PR DESCRIPTION
**Pull request by unorthodoxgeek**
removing the required flag

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -22,7 +22,7 @@ gem 'validates_lengths_from_database',  '~> 0.2'
 gem 'friendly_id', '~> 4.0'
 gem 'secure_headers', '~> 1.3'
 gem 'safemode', '~> 1.2'
-gem 'fast_gettext', '~> 0.8'
+gem 'fast_gettext', '0.9.0'
```
Comment (dLobatog) [inline Gemfile]: No need for this change now, gem was pinned already

**Comment by domcleal** [comment]
plus the v2 controller

**Comment by dLobatog** [comment]
@unorthodoxgeek ping :bell:

**Comment by unorthodoxgeek** [comment]
fixed.

**Comment by ohadlevy** [comment]
merged as 06c8a10 thanks @unorthodoxgeek

**Pull request closed without merging**